catalog/java/io/material/catalog/slider/SliderMainDemoFragment.java

```diff
@@ -37,6 +39,19 @@ public View onCreateDemoView(
         layoutInflater.inflate(R.layout.cat_slider_fragment, viewGroup, false /* attachToRoot */);
 
     Slider slider = view.findViewById(R.id.slider);
+    slider.setOnSliderTouchListener(new Slider.OnSliderTouchListener() {
+      @Override
+      public void onStartTrackingTouch(Slider slider) {
+        Snackbar.make(slider, getText(R.string.cat_slider_start_touch_description),
```

Comment:
you can just pass the resource id to this method

Reply:
Done